cfgov/jinja2/v1/sublanding-page-1/_template-prototype-full-width-content.html

```diff
@@ -0,0 +1,9 @@
+<h2>Full-width text module</h2>
+  <p>
```

Comment:
MINOR: Remove unnecessary indentation, here and throughout the rest of the file.